lib/libc/include/any-windows-any/d2d1helper.h

```diff
@@ -24,7 +24,7 @@ template<> struct TypeTraits<UINT32> {
     typedef D2D1_RECT_U    Rect;
 };
 
-static inline FLOAT FloatMax() {
+static inline FLOAT float.max() {
```

Comment:
I cannot trust this pull request because it is clear that you used a find-and-replace tool and then did not inspect the results carefully. In fact you did not even take note of which directories had files changed inside of them.

Because of this it's more work for me to review your PR than it would be for me to do the work myself.